ush/templates/FV3LAM_wflow.xml

```diff
@@ -537,6 +575,7 @@ MODULES_RUN_TASK_FP script.
 {%- endif %}
 
 {%- if do_spinup %}
+{%- if not do_ensemble %}
```

Comment:
Can you use the do_spinup variable to turn this task on/off instead of using "not do_ensemble"?

Reply:
I like it to be clear by specifying it explicitly. Less confusion, I think.

Reply:
We have too many switches on {% if not do_ensemble %}. I think we don't need to introduce extra logic if there is already a working one there. Use {% if not do_ensemble %} or {% if do_ensemble %} only when it is a must.

In the early times, I commit RTMA changes to RRFS_dev1, I added lots of {% if is_rtma %} and {% if not is_rtma %} logics. Ming @hu5970  and other would like me to remove them to keep logic in the xml file general and simple as much as possible. 

I hope we can keep the same strategy for do_ensemble or any other future specific applications.

Reply:
Removed quite a few " {% if not do_ensemble %}".